components/feature/syncedtabs/src/main/java/mozilla/components/feature/syncedtabs/controller/DefaultController.kt

```diff
@@ -45,4 +49,16 @@ internal class DefaultController(
             }
         }
     }
+
+    /**
+     * See [SyncedTabsController.syncAccount]
+     */
+    override fun syncAccount() {
+        scope.launch {
+            accountManager.withConstellation {
+                refreshDevicesAsync().await()
+            }
+            accountManager.syncNowAsync(SyncReason.User)
```

Comment:
Before this used `debounce=true`, now it's `debounce=false`. Any reason for changing this?

Reply:
Yes, it's due to this bug: https://github.com/mozilla-mobile/android-components/issues/6878